src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/HearingData.java

```diff
@@ -139,4 +139,7 @@ public class HearingData {
 
     @JsonProperty("fillingFormRenderingInfo")
     private String fillingFormRenderingInfo;
+
+    @JsonProperty("hasJudgeProvidedHearingDetails")
+    private String hasJudgeProvidedHearingDetails;
```

Comment:
Should be YesOrNo field instead of String

Reply:
Done